readme.md

```diff
@@ -16,13 +16,18 @@ We accept contributions for commonly used modern file formats, not historical or
 npm install file-type
 ```
 
-**This package is an ESM package. Your project needs to be ESM too. [Read more](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c).**
```

Comment:
This would be enough:

```
**This package is an ESM package. Your project needs to be ESM too. [Read more](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c). And for TypeScript + CommonJS, see [`load-esm`](https://github.com/Borewit/load-esm)**
```

the rest can be documented in your load-esm readme.